internal/collector/vector/visitors.go

```diff
@@ -14,7 +14,7 @@ const (
 
 func CollectorVisitor(collectorContainer *corev1.Container, podSpec *corev1.PodSpec) {
 	collectorContainer.Env = append(collectorContainer.Env,
-		corev1.EnvVar{Name: "VECTOR_LOG", Value: "info"},
+		corev1.EnvVar{Name: "VECTOR_LOG", Value: "WARN"},
```

Comment:
@vparfonov this code is common to fluentd.  can you also investigate:
* what is the impact on also changing fluentd
* does fluentd honor "WARN" or does it require "warn" ?

Reply:
- It will not impact fluentd [see here](https://github.com/openshift/cluster-logging-operator/blob/master/internal/generator/fluentd/conf.go#L55)
- fluentd require "warn" in lowercase [fluentd/log-level](https://docs.fluentd.org/deployment/logging#log-level)